README.md

```diff
@@ -113,7 +113,7 @@ npm run lint:fix
 You can run the tests with:
 
 ```sh
-vendor/bin/phpunit
+./vendor/bin/pest
```

Comment:
Is calling `./vendor/bin/pest` or `php artisan test` the correct/recommended way to execute the tests?